_includes/nav-header.html

```diff
@@ -0,0 +1,82 @@
+{% if site.navigation_header %}
+<nav class="nav  nav--header">
+  <ul class="list  list--nav">
+    {% for item in site.navigation_header %}
+
+      {% if item.url %}
+
+        {% if item.url contains '://' %}
+          {% assign url = item.url %}
+        {% else %}
+          {% assign url = item.url | relative_url %}
+        {% endif %}
+
+        <li class="item  item--nav{% if item.url == page.url %}  item--current{% endif %}">
+          <a href="{{ url }}">{{ item.title }}</a>
+        </li>
+
+      {% else %}
+
+        <li class="item  item--nav item--parent">
+          <span>{{ item.title }}</span>
+          {% if item.items %}
+            <ul class="list list--nav list--nested">
+              {% for child in item.items %}
+
+                {% if child.url contains '://' %}
+                  {% assign url = child.url %}
+                {% else %}
+                  {% assign url = child.url | relative_url %}
+                {% endif %}
+
+                <li class="item  item--nav item--child">
+                  <a href="{{ url }}">{{ child.title }}</a>
+                </li>
+
+              {% endfor %}
+            </ul>
+          {% endif %}
+        </li>
+
+      {% endif %}
+
+    {% endfor %}
+  </ul>
+  <button class="button  button--nav" aria-label="Menu toggle">
+    {% include icon.html id="nav" title="Menu" %}
+  </button>
+</nav>
+{% else %}
+  {% include nav-default.html %}
+{% endif %}
+
+<script type="text/javascript">
```

Comment:
Mostly out of curiosity, what is this javascript doing? A short comment would suffice.

Reply:
@mlubin the js does the nav toggle on mobile, hiding the nav on mobile and listening for clicks on the nav button. Jekyll lets you overwrite includes, I needed to do it to add the dropdown. Alembic has this same js in it's `nav-header.html`, so I had to copy it in too 🙂